test/commands/solution/generate_help_file_test.rb

```diff
@@ -6,7 +6,35 @@ class Solution::GenerateHelpFileTest < ActiveSupport::TestCase
     solution = create :concept_solution, exercise: exercise
 
     contents = Solution::GenerateHelpFile.(solution)
-    expected = "# Help\n\n## Running the tests\n\nRun the tests using `ruby test`.\n\n## Submitting your solution\n\nTODO: (Required) define generic submit instructions\n\n## Need to get help?\n\nTODO: (Required) define generic help text\n\nStuck? Try the Ruby gitter channel." # rubocop:disable Layout/LineLength
+    expected = <<~EXPECTED.strip
```

Comment:
I've changed the test to use a heredoc which IMO greatly helps knowing what value we are actually expecting to see returned.